openmls/src/credentials/mod.rs

```diff
@@ -184,67 +184,23 @@ pub struct Certificate {
 ///     };
 /// } Credential;
 /// ```
-#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
+#[derive(
+    Debug,
+    PartialEq,
+    Eq,
+    Clone,
+    Serialize,
+    Deserialize,
+    TlsSize,
```

Comment:
I thought I had a reason not to do it this way. But I can't recall and maybe it was only necessary in a previous iteration. Anyway, I don't see a reason not to do it this way right now.